src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceIntellisense/KustoIntellisenseHelper.cs

```diff
@@ -234,33 +197,41 @@ public static CompletionItemKind CreateCompletionItemKind(CompletionKind kustoKi
         /// <summary>
         /// Gets default keyword when user if not connected to any Kusto cluster.
         /// </summary>
-        public static LanguageServices.Contracts.CompletionItem[] GetDefaultKeywords(ScriptDocumentInfo scriptDocumentInfo, Position textDocumentPosition){
-                var kustoCodeService = new KustoCodeService(scriptDocumentInfo.Contents, GlobalState.Default);
-                var script = CodeScript.From(scriptDocumentInfo.Contents, GlobalState.Default);
-                script.TryGetTextPosition(textDocumentPosition.Line + 1, textDocumentPosition.Character, out int position);     // Gets the actual offset based on line and local offset      
-                var completion = kustoCodeService.GetCompletionItems(position);
-
-                List<LanguageServices.Contracts.CompletionItem> completions = new List<LanguageServices.Contracts.CompletionItem>();
-                foreach (var autoCompleteItem in completion.Items)
-                {
-                    var label = autoCompleteItem.DisplayText;
-                    // convert the completion item candidates into vscode format CompletionItems
-                    completions.Add(AutoCompleteHelper.CreateCompletionItem(label, label + " keyword", label, CompletionItemKind.Keyword, scriptDocumentInfo.StartLine, scriptDocumentInfo.StartColumn, textDocumentPosition.Character));
-                }
+        public static LanguageServices.Contracts.CompletionItem[] GetDefaultKeywords(
+            ScriptDocumentInfo scriptDocumentInfo, Position textDocumentPosition)
+        {
+            var kustoCodeService = new KustoCodeService(scriptDocumentInfo.Contents, GlobalState.Default);
```

Comment:
Looks like these are not recent changes. Can you please merge from main branch otherwise it will overwrite latest changes.

Reply:
Was it this specific function? The main function seems to match this from what I see here.
https://github.com/microsoft/sqltoolsservice/blob/fd4d067e711fc6f880f75bac66d49633412665a9/src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceIntellisense/KustoIntellisenseHelper.cs#L237